spec/moneta/adapters/redis/adapter_redis_spec.rb

```diff
@@ -17,4 +17,20 @@
 
     moneta_specs NATIVE_EXPIRY_SPECS.with_default_expires
   end
+
+  describe '.delete' do
+    context 'when @backend.get returns nil' do
+      let(:redis_host) { 'localhost' }
+      let(:redis_port) { '6379' }
+
+      moneta_build do
+        Moneta::Adapters::Redis.new(host: redis_host, port: redis_port, db: 6)
+      end
+
+      it 'returns nil' do
```

Comment:
The _actual_ example is when given a non-existent key, it does not raise an exception and returns nil.
Having a nil key is not necessarily the same as a non-existent actual string key.

Reply:
You mean that I should test like that?
```
      it 'returns nil' do
        expect(store.delete('non_existent_key')).to eq(nil)
      end
```